apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -777,6 +777,9 @@ type ProvisionStatus struct {
 
 	// Custom deploy procedure applied to the host.
 	CustomDeploy *CustomDeploy `json:"customDeploy,omitempty"`
+
+	// ManualCleanFinished indicates that manual clean completed successfully.
+	ManualCleanFinished bool `json:"manualCleanFinished,omitempty"`
```

Comment:
I think we need a generic solution to this, like e.g. the substates proposed in https://github.com/metal3-io/metal3-docs/blob/main/design/baremetal-operator/sub-states.md#baremetalhost-detailed-activity, and not a field specific to this one thing. I'm certain this won't be the last time we want something like this.

Also, the term 'manual cleaning' is highly specific to the implementation details of RAID in Ironic, and won't mean anything to users. We should avoid it in the API.

Reply:
Understood.